cli/args/flags.rs

```diff
@@ -405,6 +406,7 @@ pub struct Flags {
   pub reload: bool,
   pub seed: Option<u64>,
   pub unstable: bool,
+  pub unstable_features: Vec<&'static str>,
```

Comment:
I think we should just use a boolean property for each unstable flag. That will allow us to refactor the code more easily since we'll get autocompletion and type errors when removing a feature.